examples/02_single_dir/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -eoux pipefail
```

Comment:
what does this doooo; comment plz

Reply:
... https://github.com/Shemnei/punktf/pull/90#discussion_r1032774547